src/Search/SearchContext.tsx

```diff
@@ -18,6 +18,18 @@ export const useSearch = () => {
   return value
 }
 
+export const useSearchResult = ({target, searchBy, sort = true}) => {
```

Comment:
What should be the type of `target` in this case?

I tried `Array<Record<string, unknown>>`, but I am getting ts errors below when using the object

Reply:
get rid of this hook. keep it simple and filter directly in the component